README-docker-compose.md

```diff
@@ -147,7 +147,7 @@
     - `docker-compose run --rm web python manage.py migrate`
 - Populate institutions:
   - After resetting your database or with a new install you will need to populate the table of institutions. **You must have run migrations first.**
-    - `docker-compose run --rm web python -m scripts.populate_institutions test`
+    - `docker-compose run --rm web python -m scripts.populate_institutions -e test -a`
```

Comment:
I decided not to include the full usage (as shown below) since this is a setup guide that shouldn't distract users with unnecessary details.

```
usage: populate_institutions.py [-h] [-e ENV] [-i IDS [IDS ...] | -a]

optional arguments:
  -h, --help            show this help message and exit
  -e ENV, --env ENV     select the server: prod, test, stage, stage2 or stage3
  -i IDS [IDS ...], --ids IDS [IDS ...]
                        select the institution(s) to add or update
  -a, --all             add or update all institutions
```